firmware/hw_layer/digital_input/trigger/trigger_input_exti.cpp

```diff
@@ -24,9 +24,8 @@
 static ioline_t shaftLines[TRIGGER_SUPPORTED_CHANNELS];
 static ioline_t camLines[CAM_INPUTS_COUNT];
 
-static void shaft_callback(void *arg) {
+static void shaft_callback(void *arg, efitick_t stamp) {
```

Comment:
Why does this file exists at all?  Seems like the API for using EXTI or ICU should be the same; why should there be trigger specific code for exti vs others?

Reply:
if you want something to do: https://github.com/rusefi/rusefi/issues/2123